WalletWasabi/BuyAnything/BuyAnythingManager.cs

```diff
@@ -46,6 +46,11 @@ public BuyAnythingManager(string dataDir, TimeSpan period, BuyAnythingClient cli
 	{
 		Client = client;
 		FilePath = Path.Combine(dataDir, "Conversations", "Conversations.json");
+
+		BuyAnythingDataFolder = Path.Combine(EnvironmentHelpers.GetFullBaseDirectory(), "BuyAnything", "Data");
+		CountriesFilePath = Path.Combine(BuyAnythingDataFolder, "Countries.json");
+		IoHelpers.EnsureDirectoryExists(BuyAnythingDataFolder);
```

Comment:
This is no correct, we don't want to create the directory. If the directory is not there then there is a bug.